packages/react-mutation-mapper/src/component/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -560,33 +587,18 @@ export default class LollipopMutationPlot<
             '.lollipop-svgnode'
         )[0] as any;
 
-        const SVG_NS = 'http://www.w3.org/2000/svg';
         const TEXT_STYLE = 'font-size: 12px; font-family:Arial;';
         const TRACK_INDENT = 16;
         const SUBTRACK_INDENT = TRACK_INDENT + 12;
 
-        function createTextTag(textNode?: Text, x: number = 0, y: number = 0) {
-            const text = document.createElementNS(
-                SVG_NS,
-                'text'
-            ) as SVGTextElement;
-            $(text).attr({ style: TEXT_STYLE, x, y });
-            if (textNode) {
-                text.append(textNode);
-            }
-            return text;
-        }
-
-        function createGroupTag(transform?: string) {
-            const group = document.createElementNS(SVG_NS, 'g') as SVGGElement;
-            group.setAttribute('transform', transform || '');
-            return group;
-        }
-
         var outerSvg = document.createElementNS(SVG_NS, 'svg') as SVGElement;
-        outerSvg.append(lollipopSvg.cloneNode(true));
-        this.divContainer.appendChild(outerSvg);
 
+        // Make copy of chart to be used in download
+        var chartClone = lollipopSvg.cloneNode(true);
+        outerSvg.append(chartClone);
+        document.body.appendChild(outerSvg);
```

Comment:
you should put a comment here explaining why you are inserting it into DOM ("in order to measure it"). also, wrap it in a div <div class="invisible"></div> so that user doesn't see it